NAMESPACE

```diff
@@ -2,6 +2,7 @@
 
 S3method(as_model,babylon_process)
 S3method(as_model,bbi_nonmem_model)
+S3method(as_summary_list,bbi_summary_log_df)
```

Comment:
IGNORE THIS

I'm not sure what's going on here, but I'm sure I just tangled something in the rebasing. All of the branches involved have this (and the other stuff I'm saying to ignore below) so I don't understand why their showing up as changes.

https://github.com/metrumresearchgroup/rbabylon/blob/fix/rerun-example-model/NAMESPACE#L5
https://github.com/metrumresearchgroup/rbabylon/blob/fix/new-summary-fields/NAMESPACE#L5
https://github.com/metrumresearchgroup/rbabylon/blob/fix/remove-covcor/NAMESPACE#L5
https://github.com/metrumresearchgroup/rbabylon/blob/develop/NAMESPACE#L5